AzureCalling/Controllers/StartCallViewController.swift

```diff
@@ -42,6 +42,11 @@ class StartCallViewController: UIViewController {
         view.addGestureRecognizer(tap)
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
```

Comment:
Is setting the title to hide the back button? if yes, we might also try navigationItem.backBarButtonItem to nil.